src/main/webapp/i18n/de/exam.json

```
@@ -234,6 +230,8 @@
             "exerciseName": "Name",
             "noStudentExam": "Du bist nicht für die Klausur angemeldet. Bitte wende dich an die entsprechende Lehrkraft.",
             "noFurtherAttempts": "Derzeit sind keine weiteren Versuche für die Testklausur möglich. Um deine Abgabe zu betrachten, navigiere zur Klausurseite ",
+            "atLeastTutorStudentExam": "Da du kein Student bist, kannst du nicht an der Klausur teilnehmen:",
```

not sure about this one
```suggestion
            "atLeastTutorStudentExam": "Da du nicht Studierender bist, kannst du nicht an der Klausur teilnehmen:",
```